arangod/Pregel/PregelFeature.cpp

```diff
@@ -361,8 +362,11 @@ ResultT<ExecutionNumber> PregelFeature::startExecution(TRI_vocbase_t& vocbase,
 
   _actorRuntime->spawn<conductor::ConductorActor>(
       vocbase.name(),
-      std::make_unique<conductor::ConductorState>(executionSpecifications,
-                                                  vocbase),
+      std::make_unique<conductor::ConductorState>(
+          executionSpecifications,
+          std::make_unique<conductor::VocbaseLookupInfo>(
+              vocbase, executionSpecifications.vertexCollections,
+              executionSpecifications.edgeCollections)),
```

Comment:
For some reason I seem to prefer 
```suggestion
         auto vocbaseLookupInfo = std::make_unique<conductor::VocbaseLookupInfo>(
              vocbase, executionSpecifications.vertexCollections,
              executionSpecifications.edgeCollections); 
         std::make_unique<conductor::ConductorState>(
          executionSpecifications, std::move(vocbaseLookupInfo),
```